### PR TITLE
Master latest dependencies

### PR DESCRIPTION
**Pull request by jmartinm**

### CODE REVIEW

File: base.requirements.txt [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-git+https://github.com/inspirehep/invenio-search@master-es-indices#egg=invenio-search==0.1.4.dev20150904
-git+https://github.com/inspirehep/invenio-oaiharvester@master#egg=invenio-oaiharvester==0.1.2.dev20150825
-git+https://github.com/inspirehep/invenio-workflows@master#egg=invenio-workflows==0.1.2.dev20150825
-git+https://github.com/inspirehep/invenio-collections@master#egg=invenio-collections==0.1.3.dev20150904
-git+https://github.com/inspirehep/invenio-accounts@master#egg=invenio-accounts==0.1.3.dev20150903
-git+https://github.com/inspirehep/invenio-deposit@labs#egg=invenio-deposit==0.2.1.dev20150914
-git+https://github.com/inspirehep/invenio-upgrader@master#egg=invenio-upgrader==0.1.2.dev20150907
 git+https://github.com/inveniosoftware/beard@master#egg=beard
+
+## Invenio packages not yet on Pypi
```
Comment (jalavik) [inline base.requirements.txt]: best case, we can use a released version.

File: setup.py [hunk 2]
```diff
@@ -36,22 +36,27 @@
     "raven==5.0.0",  # FIXME: To be compatible with our sentry version
     "orcid",
     "retrying",
-    "invenio-query-parser>=0.3.0",
     "invenio_classifier==0.1.0",
-    "invenio-access==0.1.0",
-    "invenio-client==0.1.0",
-    "invenio-formatter==0.2.1",
-    "invenio-groups==0.1.3",
+    "invenio-access==0.2.0",
```
Comment (jalavik) [inline setup.py]: Above, in base.requirements.txt, we are using the `master` branch, so if that is needed this line should be removed.

**Comment by jmartinm** [comment]
So @jalavik @kaplun this branch is ready. Once we are ready we need to:

* [ ] Update main Docker image to be equal to `next`
* [ ] Update fork branches to be `master` instead of temporary branch
* [ ] Fix the commented out code regarding plotextractor

Something else?

**Comment by jalavik** [comment]
@jmartinm Sounds good to me. One question is if we merge existing PRs now or after?

**Comment by kaplun** [comment]
IMHOl, now that @jmartinm managed to port the overaly it's probably worth moving forward and merge on top of it's updated branch.

**Comment by jmartinm** [comment]
+1 We can merge this branch (after the small updates) and rebase the PRs.

There should not be many conflicts/changes to be made to the existing PRs, only install clean this branch first to have all the dependencies correctly installed.

**Comment by jalavik** [comment]
Sounds good. I'll look into the plotextractor issue.

**Comment by jmartinm** [comment]
@jalavik nice. So we wait and update this PR once you have the plotextractor working here? Or we merge this into `master` and then you create a PR against `master` to fix it.

**Comment by jalavik** [comment]
Indeed, it does not have to be a blocker for merging. 

Btw, this also means that `labs` on inspirehep/invenio also needs updating, I guess?

**Comment by jmartinm** [comment]
Yes, [this branch](https://github.com/inspirehep/invenio/commits/labs-latest-dependencies) will be the new labs.

**Comment by jalavik** [comment]
Merged already.

**Pull request closed without merging**